rust/Cargo.lock

```diff
@@ -273,8 +249,22 @@ checksum = "d0830ae4cc96b0617cc912970c2b17e89456fecbf55e8eed53a956f37ab50c41"
 dependencies = [
  "hmac 0.11.0",
  "pbkdf2 0.9.0",
- "rand 0.8.5",
- "sha2",
+ "rand",
+ "sha2 0.9.9",
+ "unicode-normalization",
+ "zeroize",
+]
+
+[[package]]
+name = "bip0039"
+version = "0.10.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
This reminds me of an issue from the `lib` ... https://github.com/zingolabs/zingolib/issues/8

Do we know this is the correct `BIP0039`? If not, is it good enough for now, but perhaps we should track it with an issue in this repo?

Reply:
`bip0039` is correct.

https://crates.io/crates/bip0039